.github/workflows/pypi-publish.yml

```diff
@@ -16,7 +10,12 @@ on:
 
 jobs:
   publish:
+    environment: release
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      packages: write
 
     steps:
     - uses: actions/checkout@v3
```

Comment:
Might be a good time to use the latest version (v4)
Same for the actions/setup-python further down (v5)

Reply:
Using v4 with https://github.com/ietf-tools/svgcheck/pull/50/commits/a3110f33fdeef5bae0b4f6c17e5b0a03acf32660

Reply:
Using `actions/setup-python` `v5` with https://github.com/ietf-tools/svgcheck/pull/50/commits/5693a2a0405457f890704235ea3d91363e9dab55